gui-tests/tester.js

```diff
@@ -0,0 +1,252 @@
+// This package needs to be install:
```

Comment:
how much of this is specific to docs.rs? 

skimming over it I would guess it's generic? is it the same as in rustdoc?

Reply:
It's a simplified version.

Reply:
Hopefully it might get useless in the future if I make the default runner for the framework better.